docs/suspensive.org/src/components/Sandpack/baseTemplate.ts

```diff
@@ -24,7 +24,7 @@ export default function App() {
     },
   },
   dependencies: {
-    '@tanstack/react-query': '^4.0.0',
```

Comment:
@manudeli https://github.com/toss/suspensive/pull/995#discussion_r1657561711

I'm curious if all the features of Suspensive are intended for TanStack Query version 4. 🤔
Are there any features in Suspensive that only work with TanStack Query version 5?